railties/lib/rails/generators/rails/scaffold_controller/USAGE

```diff
@@ -12,7 +12,7 @@ Description:
 Example:
     `rails generate scaffold_controller CreditCard`
 
-    Credit card controller with URLs like /credit_card/debit.
```

Comment:
Your proposed change doesn't seem clearer to me. Why don't we just fix the URL instead?